arangod/Transaction/Helpers.cpp

```diff
@@ -372,6 +372,16 @@ VPackSlice transaction::helpers::extractRevSliceFromDocument(VPackSlice slice) {
   return slice.get(StaticStrings::RevString);
 }
 
+velocypack::StringRef transaction::helpers::extractCollectionFromId(velocypack::StringRef id) {
+  std::string_view view(id.data(), id.size());
```

Comment:
no need to create a string_view here, as `velocypack::StringRef` already supports `find(char)`.